### ISSUE TITLE
[backgroundroll@Sokawaii25] Error reported in ~/.xsession-errors

### ISSUE THREAD

**Issue by claudiux** [linuxmint/cinnamon-spices-applets#3161]
```
 * Applet version/Build date: Updated on 2020-08-02 22:58:14
 * Cinnamon version: 4.4.8
 * Distribution: Mint 19.3
 * Graphics hardware *and* driver used: no matter
 ```

@Sokawaii25

Thank you for your applet!

**Issue**

This error is reported in the `~/.xsession-errors` file:

```
Cjs-Message: 03:21:04.400: JS LOG: [LookingGlass/error] (intermediate value).set_applet_label is not a function
Cjs-Message: 03:21:04.400: JS LOG: [LookingGlass/trace] 
<----------------
BackgroundRoll@/home/claude/.local/share/cinnamon/applets/backgroundroll@Sokawaii25/applet.js:32:13
main@/home/claude/.local/share/cinnamon/applets/backgroundroll@Sokawaii25/applet.js:64:12
createApplet@/usr/share/cinnamon/js/ui/appletManager.js:581:18
addAppletToPanels@/usr/share/cinnamon/js/ui/appletManager.js:356:22
finishExtensionLoad@/usr/share/cinnamon/js/ui/appletManager.js:95:14
Extension.prototype._init/<@/usr/share/cinnamon/js/ui/extension.js:277:17
---------------->
Cjs-Message: 03:21:04.405: JS LOG: [LookingGlass/info] Loaded applet backgroundroll@Sokawaii25
```

**Steps to reproduce**

Install this applet then open `~/.xsession-errors`.

**Expected behaviour**

No error reported in `~/.xsession-errors`.

**Other information**

This applet works despite this error, which appears even after restarting Cinnamon or doing a Log Out/Log In.

**Comment by claudiux** [comment]
Version 1.02: Now, a new error: in `applet.js` lines 31 to 34 must be deleted, else this applet no longer works.

**Comment by claudiux** [comment]
Fixed by #3163.
